### PR TITLE
Feature/ruby 2448 pafs bau back office add rma

### PR DESCRIPTION
**Pull request by brujeo**
https://eaflood.atlassian.net/browse/RUBY-2448
- Add RMA page

### CODE REVIEW

File: app/views/admin/organisations/edit_authority.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+TO DO
```
Comment (brujeo) [inline app/views/admin/organisations/edit_authority.html.erb:1]: These dummy templates were added as placeholders and will be implemented as part of the next task

File: app/views/admin/organisations/edit_pso.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+TO DO
```
Comment (brujeo) [inline app/views/admin/organisations/edit_pso.html.erb:1]: These dummy templates were added as placeholders and will be implemented as part of the next task

File: app/views/admin/organisations/new_authority.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+TO DO
```
Comment (brujeo) [inline app/views/admin/organisations/new_authority.html.erb:1]: These dummy templates were added as placeholders and will be implemented as part of the next task

File: app/views/admin/organisations/new_pso.html.erb [hunk 4]
```diff
@@ -0,0 +1 @@
+TO DO
```
Comment (brujeo) [inline app/views/admin/organisations/new_pso.html.erb:1]: These dummy templates were added as placeholders and will be implemented as part of the next task

File: app/controllers/admin/organisations_controller.rb [hunk 5]
```diff
@@ -21,36 +21,63 @@ def index
       @organisations = @organisations.order(name: "asc").page(page)
     end
 
+    def new
+      @organisation = Organisation.new
+      @organisation.area_type = params[:type] if params[:type].present?
```
Comment (PaulDoyle-DEFRA) [inline app/controllers/admin/organisations_controller.rb:26]: What are the implications if `area_type` is not set?

Comment (brujeo) [reply app/controllers/admin/organisations_controller.rb:26]: when area_type not set, a different template is shown - select organisation type
image

**Review by PaulDoyle-DEFRA — APPROVED** [review]

**Pull request merged**